defaults/main.yml

```diff
@@ -2,8 +2,17 @@
 # defaults file for ansible-coreos-kubelet
 coreos_kubelet_systemd: /etc/systemd/system
 coreos_kubelet_install_dir: /opt/bin
-kubelet_version: "v1.4.1_coreos.0"
+kubelet_version: "v1.7.0_coreos.0"
```

Comment:
v1.7.3_coreos.0 seems to be now available. This has a fix for https://github.com/kubernetes/kubernetes/issues/48848